### PR TITLE
fix(finance-portal-ui/issues/55): fixed decimalRate configuration for displaying forex rate

### PR DESCRIPTION
**Pull request by sivachundur**
Fix for : https://github.com/mojaloop/finance-portal-ui/issues/55

In Finance portal decimalRate configuration for displaying forex rate was not working correctly for the rates following the  different precisions due to the logic was hardcoded to handle only few scenarios, with this fix the configuration is made dynamic to handle any rate followed with any decimal precision.

### CODE REVIEW

File: src/components/SettlementWindowsTab.js [hunk 1]
```diff
@@ -317,6 +317,7 @@ function SettlementWindowsGrid(props) {
                   HUB 312 Report
                 </Button>
               </Grid>
+              /*
```
Comment (mdebarros) [inline src/components/SettlementWindowsTab.js:320]: Please add a comment as to why this is being kept.

If there is no reason to do so, then alternatively remove it please.

File: src/components/ForexRatesTab.js [hunk 2]
```diff
@@ -9,25 +9,23 @@ import SnackbarContentWrapper from './SnackbarUtils';
 import { getForexRates } from '../api';
 
 export const stringRateFromDecimalRateAndInteger = (decimalRate, integer) => {
+  const forexRate = String(integer);
```
Comment (mdebarros) [inline src/components/ForexRatesTab.js:12]: Any reason we dont use the https://github.com/mojaloop/ml-number library for this?

I believe it already has the functionality to convert integers to string, and also set the "fixed" decimals?

File: src/components/ForexRatesTab.js [hunk 3]
```diff
@@ -9,25 +9,23 @@ import SnackbarContentWrapper from './SnackbarUtils';
...
+  let precision = forexRate.length - decimalRate;
```
Comment (mdebarros) [inline src/components/ForexRatesTab.js:13]: You should be able to simplify allot of this logic doing something like this:

```
const forexRateWithPrecision = new MLNumber(`${integer}e-${precision}`) // can be instantiated with a string
return forexRateWithPrecision.toString()
```

**Review by mdebarros — CHANGES_REQUESTED** [review]
Hey @sivachundur,

A couple of comments for you, let me know what you think.

File: src/components/ForexRatesTab.js (hunk 3 above)
Comment (mdebarros) [reply src/components/ForexRatesTab.js:13]: > You should be able to simplify allot of this logic doing something like this:
> 
> ```
> const forexRateWithPrecision = new MLNumber(`${integer}e-${precision}`) // can be instantiated with a string
> return forexRateWithPrecision.toString()
> ```

As discussed, I have added a `shiftedBy` method on [ml-number/releases/tag/v11.2.0](https://github.com/mojaloop/ml-number/releases/tag/v11.2.0), which can be used as follows:

```
const ratePrecision = 4
const shifted = new MLNumber('0123').shiftedBy(-1*ratePrecision)
return shifted.toString() <-- will return '0.0123'
```

File: src/components/SettlementWindowsTab.js (hunk 1 above)
Comment (sivachundur) [reply src/components/SettlementWindowsTab.js]: Added comment.

644 report is no longer be used, more details can be found in the MES 1 Release notes https://modusbox.atlassian.net/wiki/spaces/CV/pages/2007334970/MES+1+Mowali+Release+Notes

File: src/components/ForexRatesTab.js (hunk 3 above)
Comment (sivachundur) [reply src/components/ForexRatesTab.js]: Thanks @mdebarros for adding the  'shiftedBy' function to @mojaloop/ml-number Library. 
Refactored the logic using the @mojaloop/ml-number Library

File: src/components/SettlementWindowsTab.js [hunk 4]
```diff
@@ -317,6 +317,7 @@ function SettlementWindowsGrid(props) {
                   HUB 312 Report
                 </Button>
               </Grid>
+              /* This report is no longer be used, please refer to MES 1 Release for more details https://modusbox.atlassian.net/wiki/spaces/CV/pages/2007334970/MES+1+Mowali+Release+Notes
```
Comment (mdebarros) [inline src/components/SettlementWindowsTab.js:320]:
```suggestion
              /* This report is no longer be used, but may be useful in future
```

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**